scripts/tools/eslint-plugin-prettier-internal-rules/no-empty-flat-contents-for-if-break.js

```diff
@@ -3,20 +3,21 @@
 const selector = [
   "CallExpression",
   '[callee.name="ifBreak"]',
-  "[arguments]",
+  "[arguments.length=2]",
+  ':not([arguments.0.type="SpreadElement"])',
   ">",
   ":last-child",
   '[value=""]',
```

Comment:
You know you can use `[arguments.1.`, right?

Reply:
61327b6 :+1: